### ISSUE TITLE
The 'Delimiter' option from the 'Export Messages' dialog doesn't work

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#8323]
**Storage Explorer Version**: 1.37.0-dev (103)
**Build Number**: 20241123.6
**Branch**: main
**Platform/OS**: Windows11/Windows11/Linux Ubuntu 22.04/MacOS Sequoia 15.0(Apple M1 Pro)
**Architecture**: x64/arm64/x64/arm64
**How Found**: From running test cases
**Regression From**: Not a regression

## Steps to Reproduce ##
1. Expand one storage account -> Queues.
2. Create a queue -> Click 'Export'.
3. Observe the 'Export Messages' dialog.

## Expected Experience ##
It is better to remove the 'Delimiter' option.

## Actual Experience ##
There is a 'Delimiter' option.
![Image](https://github.com/user-attachments/assets/63478599-f2e3-4045-932c-e673c5ddffd0)

## Additional Context ##
We tried exporting the message using different delimiters and the exported content remained the same.

**Comment by craxal** [comment]
@v-xianya Then it seems like the answer is not to remove to delimiter but to actually use the selected delimiter.

**Comment by craxal** [comment]
@v-xianya Sorry, let me rephrase. Export currently only outputs one column (the message), so the delimiter is effectively unused.

However, we should not remove it for a few reasons:
* Export should be symmetrical to import, where it _is_ used (even if import effectively ignores all but the first column).
* At some future date, we may import/export more columns, such a message's expiration. Already handling delimiters now will make that feature easier later.

**Comment by v-xianya** [comment]
Got it. thank you for the clarification.😊
